project.js

```diff
@@ -18,4 +19,9 @@ function CreateElement()
 	butt.setAttribute('align', 'center');
 	document.body.appendChild(butt);
 	
+}
+
+document.getElementById('kk').onkeypress = function Enterpush()
+{
+	if (event.keyCode == 13) CreateElement()

```

Comment:
Please use triple equals instead of double ones.

https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a